samples/snippets/src/main/java/com/example/bigquerystorage/JsonWriterStreamCdc.java

```diff
@@ -0,0 +1,181 @@
+
+import com.google.api.core.ApiFuture;
+import com.google.api.core.ApiFutureCallback;
+import com.google.api.core.ApiFutures;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.storage.v1.AppendRowsResponse;
+import com.google.cloud.bigquery.storage.v1.JsonStreamWriter;
+import com.google.cloud.bigquery.storage.v1.TableFieldSchema;
+import com.google.cloud.bigquery.storage.v1.TableFieldSchema.Mode;
+import com.google.cloud.bigquery.storage.v1.TableName;
+import com.google.cloud.bigquery.storage.v1.TableSchema;
+import com.google.common.util.concurrent.MoreExecutors;
+import com.google.protobuf.Descriptors.DescriptorValidationException;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+public class JsonWriterStreamCdc {
+
+  private static final String CHANGE_TYPE_PSEUDO_COLUMN = "_change_type";
+
+  private static final String CREATE_TABLE_QUERY =
+      "CREATE TABLE `%s.%s` (\n"
+          + " Customer_ID INT64 PRIMARY KEY NOT ENFORCED,\n"
+          + " Customer_Enrollment_Date DATE,\n"
+          + " Customer_Name STRING,\n"
+          + " Customer_Address STRING,\n"
+          + " Customer_Tier STRING,\n"
+          + " Active_Subscriptions JSON)\n"
+          + "OPTIONS(max_staleness = INTERVAL 15 MINUTE);";
+
+  public static void main(String[] args) throws Exception {
+    if (args.length != 4) {
+      System.out.println("Arguments: project, dataset, table, data_file");
+      return;
+    }
+
+    String projectId = args[0];
+    String datasetName = args[1];
+    String tableName = args[2];
+    String dataFile = args[3];
+
+    createDestinationTable(projectId, datasetName, tableName);
+    JSONArray records = getRecordsFromDataFile(dataFile);
+    writeToDefaultStream(projectId, datasetName, tableName, records);
+  }
+
+  public static void createDestinationTable(
+      String projectId, String datasetName, String tableName) {
+    BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+    QueryJobConfiguration queryConfig =
+        QueryJobConfiguration.newBuilder(String.format(CREATE_TABLE_QUERY, datasetName, tableName))
+            .build();
+    try {
+      bigquery.query(queryConfig);
+    } catch (BigQueryException | InterruptedException e) {
+      System.out.println("Query did not run \n" + e.toString());
+    }
+  }
+
+  // writeToDefaultStream: Writes records from the source file to the destination table.
+  public static void writeToDefaultStream(
+      String projectId, String datasetName, String tableName, JSONArray data)
+      throws DescriptorValidationException, InterruptedException, IOException {
+    // Build the table schema with an additional _change_type column.
```

Comment:
Lets call this out in a more noticeable manner, because this is the important bit in this sample.